code/game/objects/items/devices/radio/headset.dm

```diff
@@ -165,6 +165,11 @@
 			to_chat(user, SPAN_NOTICE("This headset doesn't have any encryption keys!  How useless..."))
 
 	if(istype(W, /obj/item/device/encryptionkey/))
+		for (var/obj/item/device/encryptionkey/key in keys)
```

Comment:
```suggestion
		for (var/obj/item/device/encryptionkey/key as anything in keys)
```
I haven't looked at the code but if the only type in keys are of type /obj/item/device/encryptionkey you can use as anything, it's just a micro opt.

Reply:
It's been a minute since I touched dm but irrc there is probably a macro to check if types are equal instead of x.type == y.type like is_type.

Reply:
Also approving this change. 

Thanks for the macro check tip, changing that section too.